spec/support/factory_bot.rb

```diff
@@ -0,0 +1,5 @@
+require 'factory_bot'
+
+RSpec.configure do |config|
+  config.include FactoryBot::Syntax::Methods
```

Comment:
Good stuff. You can do the same for your cucumber configuration, and make use of a shorter syntax (meaning you can omit `FactoryBot` - see example above.)

Reply:
Not sure what you mean! remove ``` require 'factory_bot``` ?

Reply:
Well, now that you mention it...
But no, I mean that in your tests, you can omit the word `FactoryBot` (as I showed in an example above) 
But out of curiosity, since you have to ask, why did you guys add this setting in the first place?